app/jinja_filters.py

```diff
@@ -246,105 +148,6 @@ def format_date_range(context, start_date, end_date=None):
-    :returns (str): The formatted range.
-    """
-    start_datetime = convert_to_datetime(start_date)
-    end_datetime = convert_to_datetime(end_date)
-
-    first_date_format = date_format
-
-    if start_datetime.year == end_datetime.year:
-        first_date_format = date_format.replace('YYYY', '')
-
-        if start_datetime.month == end_datetime.month:
-            first_date_format = first_date_format.replace('MMMM', '')
-
-    # Cleanup any extra spaces in the new format string
-    first_date_format = first_date_format.replace('  ', ' ').strip()
-
-    if not first_date_format:
-        # If the date format was entirely removed, leave it alone
-        first_date_format = date_format
-
-    output = flask_babel.gettext('%(from_date)s to %(to_date)s',
-                                 from_date=format_date_custom(context, start_date, first_date_format),
-                                 to_date=format_date_custom(context, end_date, date_format))
-
-    return mark_safe(context, output)
-
-
-def _get_first_non_empty_item(items):
-    """
-    :param items: anything that is iterable
-    :return: first non empty value
-
-    In this filter the following values are considered non empty:
-    - None
-    - any empty sequence, for example, '', (), [].
-    - any empty mapping, for example, {}.
-
-     Note: to guarantee the returned element is actually the first non empty element in the iterable,
-    'items' must be a data structure that preserves order, ie tuple, list etc.
-    If order is not important, this can be reused to return `one of` the elements which is non empty.
-
-    This filter will treat zero of any numeric type for example, 0, 0.0, 0j and boolean 'False'
-    as a valid item since they are naturally 'falsy' in Python but not empty.
-
-    Note: Booleans are a subtype of integers. Zero of any numeric type 'is not False' but 'equals False'.
-    Reference: https://docs.python.org/release/3.4.2/library/stdtypes.html?highlight=boolean#boolean-values
-    """
-    for item in items:
-        if item or item is False or item == 0:
-            return item
-
-    raise Exception('No valid items provided.')
-
-
-@evalcontextfunction
-def first_non_empty_item(context, *items):
-    """
-    :param items: tuple of items
-    :return: mark safe version of the first non empty item
-    """
-    non_empty_item = _get_first_non_empty_item(items)
-    return mark_safe(context, non_empty_item)
-
-
-@blueprint.app_template_filter()
-def format_household_name(names):
-    return concatenated_list(list_items=names, delimiter=' ')
-
-
-@blueprint.app_template_filter()
-def format_household_name_possessive(names):
-    name = format_household_name(names)
-    if name:
-        last_char = name[-1:]
-        if last_char.lower() == 's':
-            return name + '’'
-
-        return name + '’s'
-
-
 @blueprint.app_template_filter()
 def answer_is_type(answer, target_answer_type):
```

Comment:
This can go?

Reply:
This is still used in some of the summary templates